kind/pkg/exec/exec.go

```diff
@@ -87,6 +87,10 @@ func (cmd *Cmd) runLoggingOutputOnFail() error {
 // except instead of returning the byte buffer of stderr + stdout,
 // it scans these for lines and returns a slice of output line strings
 func (cmd *Cmd) CombinedOutputLines() (lines []string, err error) {
+	if cmd.Debug {
```

Comment:
Optional: Probably try different verbosity level with -v

Reply:
at this point I intend to take a stab at ripping out glog entirely, the logging situation is poor. when I do I will also refactor the logging in this package. for now I'll just continue to Info(..) these when Debug is set.